### PR TITLE
Insane in the Membrane Fan Tribute

### PR DESCRIPTION
**Pull request by ghost**
- Special thanks to @9600bauds for making #8096 
- This is a step towards #8220 as it will make variables like weakened and stunned easier to deal with.

### CODE REVIEW

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -1587,21 +1587,22 @@ var/list/slot_equipment_priority = list( \
 		stunned = max(stunned + amount,0)
 	return
 
-/mob/proc/Weaken(amount)
-	if(status_flags & CANWEAKEN)
-		weakened = max(max(weakened,amount),0)
+/mob/proc/Knockdown(amount)
```
Comment (ghost) [inline code/modules/mob/mob.dm]: whoops I wanted to discard this change

**Comment by 9600bauds** [comment]
yes
yesss
burn it all down

File: code/modules/mob/living/carbon/zombie/zombie.dm [hunk 2]
```diff
@@ -125,7 +125,7 @@
 			for (var/mob/living/carbon/C in range(viewrange-2,src.loc))
 				if (C.stat == 2 || !can_see(src,C,viewrange) || istype(C, /mob/living/carbon/human/zombie) || istype(C, /mob/living/carbon/monkey))
 					continue
-				if(C:stunned || C:paralysis || C:weakened)
+				if(C.incapacitated())
 					target = C
 					break
 				if(C:health < last_health)
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/zombie/zombie.dm:131]: fix it

Comment (ghost) [inline code/modules/mob/living/carbon/zombie/zombie.dm:131]: I have no idea what I am to fix or what the problem is *shrug

Comment (Iamgoofball) [inline code/modules/mob/living/carbon/zombie/zombie.dm:131]: look with your eyes

Comment (PJB3005) [inline code/modules/mob/living/carbon/zombie/zombie.dm:131]: Goof is talking about the usage of the : operator.

Comment (Exxion) [inline code/modules/mob/living/carbon/zombie/zombie.dm:131]: Why the fuck is it even using :? It's already typecast.

**Pull request merged**